### PR TITLE
[WIP] update: Mention `mutate` in useOrganizationList

### PR DESCRIPTION
**Pull request by panteliselef**
Please don't merge until the following is in prod release
https://github.com/clerkinc/javascript/pull/1745

### CODE REVIEW

File: docs/references/react/use-organization-list.mdx [hunk 1]
```diff
@@ -49,6 +51,19 @@ const JoinedOrganizationList = () => {
       >
         Load more
       </button>
+
+      <button
+        disabled={!isLoaded}
+        onClick={async () => {
+          await createOrganization({
+              name: 'randomOrg' + Date.now(),
+              slug: 'random-org' + Date.now(),
```
Comment (chanioxaris) [inline docs/references/react/use-organization-list.mdx:60]: We can omit slug here as it's optional and we are looking to phase it out as we are moving forward

File: docs/references/react/use-organization-list.mdx [hunk 2]
```diff
@@ -92,6 +107,19 @@ const JoinedOrganizationList = () => {
       >
         Load more
       </button>
+        
+      <button
+        disabled={!isLoaded}
+        onClick={async () => {
+          await createOrganization({
+              name: 'randomOrg' + Date.now(),
+              slug: 'random-org' + Date.now(),
```
Comment (chanioxaris) [inline docs/references/react/use-organization-list.mdx:116]: Same here about the slug

File: docs/references/react/use-organization-list.mdx [hunk 3]
```diff
@@ -11,14 +11,16 @@ The `useOrganizationList` hook allows you to retrieve the memberships, invitatio
 
 The `useOrganizationList` hook requires developers to opt-in by specifying which resource they need to fetch and paginate through. It is ideal for infinite lists or tables.
 
+Under the hood, `useOrganizationList` is powered by SWR in order to support caching and pagination. Part of the return values from this hook are directly exported from useSWR and useSWRInfinite and you can find more information in [their documentation page](https://swr.vercel.app/docs/getting-started)
```
Comment (BRKalow) [inline docs/references/react/use-organization-list.mdx:14]:
```suggestion
Under the hood, `useOrganizationList` is powered by SWR in order to support caching and pagination. Part of the return values from this hook are directly exported from `useSWR` and `useSWRInfinite`, you can find more information in the [SWR documentation](https://swr.vercel.app/docs/getting-started)
```

File: docs/references/react/use-organization-list.mdx [hunk 4]
```diff
@@ -263,3 +291,4 @@ The `userMemberships`, `userInvitations`, and `userSuggestions` are returned as
 | `fetchNext` | A helper function that triggers the previous page to be loaded. This is the same as `fetchPage(page=> Math.max(0, page + 1))` |
 | `hasNextPage` | A boolean that indicates if there are available pages to be fetched. |
 | `hasPreviousPage` | A boolean that indicates if there are available pages to be fetched. |
+| `mutate` | A function for mutating the related cache for each paginated list. Also know as [Bound Mutate](https://swr.vercel.app/docs/mutation#bound-mutate). For more details about mutations please visit the [SWR docs](https://swr.vercel.app/docs/mutation)
```
Comment (BRKalow) [inline docs/references/react/use-organization-list.mdx:294]:
```suggestion
| `mutate` | A function for mutating the related cache for each paginated list. Also known as [Bound Mutate](https://swr.vercel.app/docs/mutation#bound-mutate). For more details about mutations please visit the [SWR docs](https://swr.vercel.app/docs/mutation)
```

File: docs/references/react/use-organization-list.mdx (hunk 3 above)
Comment (alexisintech) [inline docs/references/react/use-organization-list.mdx:14]:
```suggestion
Under the hood, `useOrganizationList` is powered by SWR in order to support caching and pagination. Part of the return values from this hook are directly exported from `useSWR` and `useSWRInfinite`. You can find more information in the [SWR documentation](https://swr.vercel.app/docs/getting-started).
```

File: docs/references/react/use-organization-list.mdx (hunk 1 above)
Comment (alexisintech) [inline docs/references/react/use-organization-list.mdx:60]:
```suggestion
```

File: docs/references/react/use-organization-list.mdx (hunk 2 above)
Comment (alexisintech) [inline docs/references/react/use-organization-list.mdx:116]:
```suggestion
```

File: docs/references/react/use-organization-list.mdx (hunk 4 above)
Comment (alexisintech) [inline docs/references/react/use-organization-list.mdx:294]:
```suggestion
| `mutate` | A function for mutating the related cache for each paginated list. Also known as [Bound Mutate](https://swr.vercel.app/docs/mutation#bound-mutate). You can find more information about mutations in the [SWR documentation](https://swr.vercel.app/docs/mutation).
```

**Review by alexisintech — CHANGES_REQUESTED** [review]

**Comment by panteliselef** [comment]
I am closing this. I will create a new PR in the coming weeks that will be up to date.

**Pull request closed without merging**